fdbclient/BlobGranuleReader.actor.cpp

```diff
@@ -256,7 +256,9 @@ static void applyDeltas(std::map<KeyRef, ValueRef>* dataMap,
 ACTOR Future<RangeResult> readBlobGranule(BlobGranuleChunkRef chunk,
                                           KeyRangeRef keyRange,
                                           Version readVersion,
-                                          Reference<BackupContainerFileSystem> bstore) {
+                                          Reference<BackupContainerFileSystem> bstore,
+                                          Optional<BlobWorkerStats *> stats) {
```

Comment:
Add a comment to clean this up (the BlobGranuleReader doesn't need all these stats)